Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ruby:2.7.6
+
+
+# install rails dependencies
+RUN apt-get clean all && apt-get update -qq && apt-get install -y build-essential libpq-dev \
+    curl gnupg2 apt-utils default-libmysqlclient-dev git libcurl3-dev cmake \
+    libssl-dev pkg-config openssl imagemagick file nodejs yarn
+
+
+RUN mkdir /app
+WORKDIR /app
+
+# Adding gems
+COPY Gemfile Gemfile
```

Comment:
Why copy these files separately when you can copy the whole . folder into /app in L21?

Reply:
This is so that we don't execute `bundle install` every time we start the container since not all changes require rebuilding gems. (Take advantage of docker's caching mechanism)